fre/make/tests/null_example/null_model.yaml

```diff
@@ -0,0 +1,15 @@
+# reusable variables
+fre_properties:
+  - &VERSION        "full"
+  - &FRE_STEM       !join [null_model/, *VERSION]
+  # compile information
+  - &branch              "main"
+  - &INTEL               "intel-classic"
+  - &FMSincludes         "-IFMS/include"
```

Comment:
Any `momIncludes` needed?

Reply:
No i don't think so. I'm not sure how different ocean_null is from mom but it didn't seem to need includes when i compiled this.